src/main/java/com/minecolonies/coremod/items/ItemSupplyCampDeployer.java

```diff
@@ -108,7 +108,7 @@ public EnumActionResult onItemUse(
             fillChest((TileEntityChest) worldIn.getTileEntity(pos.up()));
 
             ItemStackUtils.changeSize(stack, -1);
-            player.addStat(ModAchievements.achievementGetSupply);
+            //player.addStat(ModAchievements.achievementGetSupply);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)